app/models/chapters.js

```diff
@@ -45,6 +45,9 @@ module.exports = (sequelize, DataTypes) => {
           allowNull: false,
           minimum: 0,
           field: 'indexinstory'
+        },
+        view_custom: {
+          type: DataTypes
```

Comment:
can you type the object more precisely?

Reply:
Sure, sorry I add it. Maybe, in other PR, I will check because I copy paste the images.iif_data element for this so maybe, others haven't type

Reply:
I meant to type the object parameters: transparency,  showBuilding, buildingsSlider, depthSlider